src/Confluent.Kafka/Producer.cs

```diff
@@ -86,7 +89,17 @@ private Task StartPollTask(CancellationToken ct)
                         while (true)
                         {
                             ct.ThrowIfCancellationRequested();
-                            ownedKafkaHandle.Poll((IntPtr)cancellationDelayMaxMs);
+                            int eventsServedCountLocal = ownedKafkaHandle.Poll((IntPtr)cancellationDelayMaxMs);
```

Comment:
i'll change this name to `eventsServedCount_`. Local postfix is a little unusual.